x-pack/plugins/actions/server/create_execute_function.ts

```diff
@@ -171,15 +177,17 @@ export function createBulkExecutionEnqueuerFunction({
           executionId: actionToExecute.executionId,
           consumer: actionToExecute.consumer,
           relatedSavedObjects: relatedSavedObjectWithRefs,
+          ...(actionToExecute.apiKeyId ? { apiKeyId: actionToExecute.apiKeyId } : {}),
```

Comment:
Storing `apiKeyId` in the `action_task_param` saved object so that the API key invalidation task can query for any scheduled actions that are using invalidated API keys and wait before performing invalidation.